File: samples/SelfAspNet/SelfAspNet/Chap10/ChartParts.ascx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace SelfAspNet.Chap10
{
    public partial class ChartParts : System.Web.UI.UserControl
    {
        public String Brand
        {
            get
            {
                return sds.SelectParameters["brand"].DefaultValue;
            }

            set
            {
                sds.SelectParameters["brand"].DefaultValue = value;
            }
        }

        protected void Page_Load(object sender, EventArgs e)
        {

        }
    }
}

File: samples/SelfAspNetVb/SelfAspNetVb/Chap10/ChartParts.ascx.vb
Public Class ChartParts
    Inherits System.Web.UI.UserControl

    Public Property Brand() As String
        Get
            Return sds.SelectParameters("brand").DefaultValue
        End Get
        Set(ByVal value As String)
            sds.SelectParameters("brand").DefaultValue = value
        End Set
    End Property
    Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load

    End Sub

End Class